docs/ref/models/instances.txt

```diff
@@ -854,6 +855,19 @@ Other attributes
     you to catch a particular model class with ``try/except``. The exception is
     a subclass of :exc:`django.core.exceptions.ObjectDoesNotExist`.
 
+``_meta``
+----------
+
+.. attribute:: Model._meta
+
+    The ``_meta`` attribute gives access to an ``Options`` object that allows
+    inspection of the fields on the model. See :doc:`/ref/models/meta`.
+
+    .. note::
+
+        Whilst an underscore prefix is typically used to indicate a private
+        attribute, the documented parts of ``_meta`` are a public, stable API.
+
```

Comment:
`_meta` has a separate page in docs and it's not really for an instance but for a model class, adding it here is misleading, IMO. I would revert these docs.

Reply:
The DoesNotExist class attribute is documented above, so this seems to be the place to document it. Nothing links to the meta page atm so it might be a bit undiscoverable without this.

Reply:
> Nothing links to the meta page atm so it might be a bit undiscoverable without this.

That's not true, see ["Field attribute reference"](https://docs.djangoproject.com/en/3.0/ref/models/fields/#field-attribute-reference). I really believe that instance docs are not the right place for `Model._meta` :shrug:

Reply:
You're right. I missed that with bad grepping. I guess it's also not the place for `DoesNotExist` either, I'll try fix that in #13194 .